src/main/java/appeng/api/implementations/blockentities/IChestOrDrive.java

```diff
@@ -62,4 +63,9 @@ public interface IChestOrDrive extends IActionHost {
     @Nullable
     Item getCellItem(int slot);
 
+    /**
+     * Returns the inventory of the storage cell in the given slot or null.
```

Comment:
Note that interacting with the inventory directly will bypass the chest/drive monitoring and thus not properly update LED colors etc. So I'd call the method `getInternalCellInventory` and add this disclaimer to the javadoc.